File: 2학년 1학기/자료구조/5장/전위표기법 트리/Node.java
public class Node<E> {
	private E item ;
	private Node<E> next;
	int isp;
	int icp;
	
	public Node(E newitem, Node<E> node,int newisp, int newicp) {
		item = newitem;
		next = node;
		isp = newisp;
		icp = newicp;
	}
	
	public E getItem() {return item;}
	public Node<E> getNext() {return next;}
	public int getisp() {return isp;}
	public int geticp() {return icp;}
	public void setItem(E newItem) {item = newItem;}
	public void setNext(Node<E> newNext) { next = newNext;}
	public void setisp(int newisp) {isp = newisp;}
	public void seticp(int newicp) {icp = newicp;}
}

File: 2학년 2학기/c++ 프로그래밍/11주차 과제/Node.cpp
#include "Node.h"

template <typename T>
Node<T>::Node(const T data, Node* next) {
	this->data = data;
	this->next = next;
}

template <typename T>
T Node<T>::getdata() {
	return data;
}

template <typename T>
Node<T>* Node<T>::getnext() {
	return next;
}

template <typename T>
void Node<T>::setdata(T d) {
	data = d;
}

template <typename T>
void Node<T>::setnext(Node<T>* n) {
	next = n;
}